data/data/vsphere/variables-vsphere.tf

```diff
@@ -0,0 +1,107 @@
+//////
+// vSphere variables
+//////
+
+variable "vsphere_url" {
+  type        = string
+  description = "This is the vSphere server for the environment."
+}
+
+variable "vsphere_username" {
+  type        = string
+  description = "vSphere server user for the environment."
+}
+
+variable "vsphere_password" {
+  type        = string
+  description = "vSphere server password"
+}
+
+variable "vsphere_cluster" {
+  type        = string
+  description = "This is the name of the vSphere cluster."
+}
+
+variable "vsphere_datacenter" {
+  type        = string
+  description = "This is the name of the vSphere data center."
+}
+
+variable "vsphere_datastore" {
+  type        = string
+  description = "This is the name of the vSphere data store."
+}
+
+variable "vsphere_resource_pool" {
+  type        = string
+  description = "This is the name of the vSphere resource pool."
+}
+
+variable "vsphere_template" {
+  type        = string
+  description = "This is the name of the VM template to clone."
+}
+
+variable "vsphere_network" {
+  type        = string
+  description = "This is the name of the publicly accessible network for cluster ingress and access."
+}
+
+variable "vsphere_folder" {
+  type = string
+}
+
+/////////
+// OpenShift cluster variables
+/////////
+
+variable "base_domain" {
+  type        = string
+  description = "The base DNS zone to add the sub zone to."
+}
+
+variable "cluster_id" {
+  type        = string
+  description = "This cluster id must be of max length 27 and must have only alphanumeric or hyphen characters."
+}
+
+variable "cluster_domain" {
+  type        = string
+  description = "The base DNS zone to add the sub zone to."
+}
+
+variable "machine_cidr" {
+  type = string
+}
+
+/////////
+// Bootstrap machine variables
+/////////
+
+variable "ignition_bootstrap" {
+  type = string
+}
+
+///////////
+// Control Plane machine variables
+///////////
+
+variable "ignition_master" {
+  type = string
+}
```

Comment:
almost all of them are already available using https://github.com/openshift/installer/blob/master/data/data/config.tf

Reply:
ack...will remove.